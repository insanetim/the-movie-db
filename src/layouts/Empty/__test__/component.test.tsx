import { render } from '@testing-library/react'

import EmptyLayout from '../component'

describe('EmptyLayout component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<EmptyLayout />)

    expect(asFragment()).toMatchSnapshot()
  })
})
