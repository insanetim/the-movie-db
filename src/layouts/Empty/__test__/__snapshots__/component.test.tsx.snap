// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyLayout component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="center"
  >
    <div
      class="ant-layout css-dev-only-do-not-override-j9bb5n"
    >
      <main
        class="ant-layout-content css-dev-only-do-not-override-j9bb5n"
      />
    </div>
  </div>
</DocumentFragment>
`;
